.travis.yml

```diff
@@ -0,0 +1,39 @@
+# Travis configuration file using the build matrix feature
+# Read more under http://docs.travis-ci.com/user/build-configuration/
+# THIS SCRIPT IS SUPPOSED TO BE AN EXAMPLE. MODIFY IT ACCORDING TO YOUR NEEDS!
+
+sudo: false
+language: python
+virtualenv:
+  system_site_packages: false
+matrix:
+  fast_finish: true
+  include:
+    - python: 3.6
```

Comment:
I think we also want to test building and running on python3.7, but I can set that up in a different PR once this is merged in and we have proper tests to run.

Reply:
Yeah... the whole .travis.yml is complete trash -- I figure you'll fix that ;-)

Reply:
Fair enough :)